ONSdigital/eq-author-app
# Ear 781 review dependabots

### What is the context of this PR?

Loooooootttsss of dependabots. Some breaking changes have been fixed; others that have them haven't as I'll do them at a later date as they're a bit more involved.

### How to review 

Run the tests in each part of the app and make sure everything passes. Try making a basic questionnaire and use common functionality to make sure it's all working.

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
